### PR TITLE
Update frontend dependencies

### PR DESCRIPTION
**Pull request by SeppPenner**
Summary of the changes (in less than 80 chars)

 * Updated npm packages.

This error occured after updating: 
grafik

I'm not an expert for React, so I have no idea :D

### CODE REVIEW

**Comment by loic-sharma** [comment]
Hm that error is weird. You should be able to get around it by changing this line:

https://github.com/loic-sharma/BaGet/blob/63d7633c070bcae5fd4faea3415568b170afe70e/src/BaGet.UI/src/Upload.tsx#L58

To:

```tsx
{'>' + value}
```

File: src/BaGet.UI/package.json [hunk 1]
```diff
@@ -13,11 +13,11 @@
   "dependencies": {
     "bootstrap": "3.3.7",
```
Comment (loic-sharma) [inline src/BaGet.UI/package.json:14]: Could you bump `bootstrap` to 3.4.1 to address this: https://github.com/advisories/GHSA-9v3m-8fp8-mj99

This should do the trick: ` npm install bootstrap@3.4.1`

Comment (SeppPenner) [reply src/BaGet.UI/package.json:14]: Ok.

**Comment by SeppPenner** [comment]
There are some more outdated packages `npm --list outdated`:

grafik

Should I update any of them as well?

**Comment by SeppPenner** [comment]
This makes https://github.com/loic-sharma/BaGet/pull/274 not needed any longer, by the way.

**Comment by loic-sharma** [comment]
Thanks for the contribution! :)

**Pull request merged**